app/javascript/dashboard/routes/dashboard/settings/sla/components/SLABusinessHoursLabel.vue

```diff
@@ -0,0 +1,34 @@
+<script setup>
```

Comment:
This is good for now. We can extract a base component from this, but that would be over-optimizing at this point. Let's keep it in mind though, the next time we build something similar, we can extract a base component out

Reply:
Yeah, sure